tests/__init__.py

```diff
@@ -24,6 +24,9 @@
 
 IS_TRAVIS = os.getenv("CI") and os.getenv("TRAVIS")
 
+# WikiMedia rejects requests without User-Agent
```

Comment:
`requests` actually sets an Uger-Agent by default, but WikiMedia probably rejects what it thinks is not a browser.

Reply:
You're right:
```
>>> import requests
>>> requests.utils.default_headers() 
{'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive'}
```
I have updated the comment and commit message.